diff --git a/node_modules/@pothos/plugin-prisma/lib/schema-builder.js b/node_modules/@pothos/plugin-prisma/lib/schema-builder.js
index 51ada45..d568f9c 100644
--- a/node_modules/@pothos/plugin-prisma/lib/schema-builder.js
+++ b/node_modules/@pothos/plugin-prisma/lib/schema-builder.js
@@ -150,7 +150,7 @@ schemaBuilderProto.prismaNode = function prismaNode(type, { id: { field, resolve
                 }
             }));
             (0, _core.brandWithType)(record, typeName);
-            return record;
+            return record ?? options.extensions.resolveWhenNull?.(rawFindUnique(id, context)) ?? null;
         }
     };
     const ref = this.prismaObject(type, extendedOptions);
