diff --git a/node_modules/cron-parser/lib/parser.js b/node_modules/cron-parser/lib/parser.js
index 362a416..83ed7b3 100644
--- a/node_modules/cron-parser/lib/parser.js
+++ b/node_modules/cron-parser/lib/parser.js
@@ -95,22 +95,4 @@ CronParser.parseString = function parseString (data) {
   return response;
 };
 
-/**
- * Parse crontab file
- *
- * @public
- * @param {String} filePath Path to file
- * @param {Function} callback
- */
-CronParser.parseFile = function parseFile (filePath, callback) {
-  require('fs').readFile(filePath, function(err, data) {
-    if (err) {
-      callback(err);
-      return;
-    }
-
-    return callback(null, CronParser.parseString(data.toString()));
-  });
-};
-
 module.exports = CronParser;
