diff --git a/node_modules/prisma/build/index.js b/node_modules/prisma/build/index.js
index 99ec67e..499d623 100755
--- a/node_modules/prisma/build/index.js
+++ b/node_modules/prisma/build/index.js
@@ -120301,6 +120301,9 @@ function simpleDebounce(fn2) {
   };
 }
 
+var performance = {
+  now: () => new Date().getTime(),
+};
 // src/Generate.ts
 var pkg = eval(`require('../package.json')`);
 var _Generate = class _Generate {
